examples/light_ws2812/Makefile

```diff
@@ -0,0 +1,28 @@
+# name of the application
+APPLICATION = light_ws2812
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= nucleo-g070rb
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+# required packages
+USEPKG += light_ws2812
+USEMODULE += xtimer
+USEMODULE += random
+
+# Comment this out to disable code in RIOT that does safety checking
+# which is not needed in a production environment but helps in the
+# development process:
+DEVELHELP ?= 1
+
+# Change this to 0 show compiler invocation lines by default:
+QUIET ?= 1
+
+CFLAGS += -DLIGHT_WS2812_UC_STM32G0XX
+
+CFLAGS += -DLIGHT_WS2812_GPIO_PIN=7
+CFLAGS += -DLIGHT_WS2812_GPIO_PORT=GPIOA
```

Comment:
I wonder if it would make sense to provide an `light_ws2812_params.h` e.g. similar to https://github.com/RIOT-OS/RIOT/blob/master/drivers/sht3x/include/sht3x_params.h

The main advantage would be that there would always be a valid fall-back configuration (so that the CI can compile applications using the pkg) and the macros could be documented there and would pop up in Doxygen. An application could provide its own `light_ws2812_params.h` in the application folder (the build system makes sure that the application folder is searched first for header files, so that one can rely on their own params overwriting the defaults).